arch/arm64/configs/gki_defconfig

```diff
@@ -46,8 +46,8 @@ CONFIG_SLAB_FREELIST_RANDOM=y
 CONFIG_SLAB_FREELIST_HARDENED=y
 CONFIG_SHUFFLE_PAGE_ALLOCATOR=y
 CONFIG_PROFILING=y
-CONFIG_ARCH_SUNXI=y
-CONFIG_ARCH_HISI=y
+#CONFIG_ARCH_SUNXI=y
```

Comment:
Kernel configuration process is like:
--> gki_defconfig --> overlay by qti configs (made by qualcom or Lantronix) --> vendor specific configs (made by Aeroratech).

It would be a better idea to escalate our configs in the vendor specific config files for better maintains. Take this commit as an example:
diff --git a/arch/arm64/configs/vendor/kona_vendor_debug.config b/arch/arm64/configs/vendor/kona_vendor_debug.config
index 53e35facdaa9..186dbcf38e0a 100644
--- a/arch/arm64/configs/vendor/kona_vendor_debug.config
+++ b/arch/arm64/configs/vendor/kona_vendor_debug.config
@@ -3,3 +3,5 @@ CONFIG_RB5_GPIOS_ENABLE=y
 CONFIG_DRM_ITE_IT6161=y
 CONFIG_DRM_FAILSAFE_BRIDGE=y
 CONFIG_INPUT_BHI160B=y
+# CONFIG_ARCH_SUNXI is not set
+# CONFIG_ARCH_HISI is not set

BTW, #CONFIG_ARCH_HISI=y is not the right way for kernel configuration, to explicitly disable an option, it should go like:
# CONFIG_ARCH_HISI is not set
Otherwise, kernel config subsystem will take it "undefined" and will use the default value as programmed in the kernel config file.